server/app/durablejobs/jobs/FixApplicantDobDataPathJob.java

```diff
@@ -0,0 +1,56 @@
+package durablejobs.jobs;
+
+import com.google.common.base.Preconditions;
+import durablejobs.DurableJob;
+import io.ebean.DB;
+import io.ebean.Database;
+import java.time.LocalDate;
+import models.Applicant;
+import models.PersistedDurableJob;
+import services.WellKnownPaths;
+import services.applicant.ApplicantData;
+
+/**
+ * Finds applicant data objects with applicant_date_of_birth set to a number and changes it to a
+ * nest object.
+ */
+public final class FixApplicantDobDataPathJob extends DurableJob {
+  private final Database database;
+
+  private final PersistedDurableJob persistedDurableJob;
+
+  public FixApplicantDobDataPathJob(PersistedDurableJob persistedDurableJob) {
+    this.persistedDurableJob = Preconditions.checkNotNull(persistedDurableJob);
+    this.database = DB.getDefault();
+  }
+
+  @Override
+  public PersistedDurableJob getPersistedDurableJob() {
+    return persistedDurableJob;
+  }
+
+  @Override
+  public void run() {
+    String sql =
+        "select * from applicants where ((object #>>"
```

Comment:
Hi Rocky- Here is the formatted query - "SELECT *
FROM applicants
WHERE ((object #>> '{}')::jsonb)::json @> '{applicant, applicant_date_of_birth}' IS NOT NULL"

Reply:
Ah ok. This version works, but is that just for readability?

Reply:
done